src/PureCloud.Client/Apis/WorkforceManagementApi.cs

```diff
@@ -0,0 +1,244 @@
+
+        return await response.Content.ReadAsStringAsync(cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task<string> GetScheduleAsync(string businessUnitId, string weekId, string scheduleId, string expand = null, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(businessUnitId);
+        ArgumentException.ThrowIfNullOrEmpty(weekId);
+        ArgumentException.ThrowIfNullOrEmpty(scheduleId);
+
+        var parameters = new NameValueCollection();
+
+        if (!string.IsNullOrEmpty(expand))
+        {
+            parameters.Add("expand", UriHelper.ParameterToString(expand));
+        }
+
+        var uri = UriHelper.GetUri($"api/v2/workforcemanagement/businessunits/{businessUnitId}/weeks/{weekId}/schedules/{scheduleId}", parameters);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadAsStringAsync(cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task DeleteScheduleAsync(string businessUnitId, string weekId, string scheduleId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(businessUnitId);
+        ArgumentException.ThrowIfNullOrEmpty(weekId);
+        ArgumentException.ThrowIfNullOrEmpty(scheduleId);
+
+        var uri = UriHelper.GetUri($"api/v2/workforcemanagement/businessunits/{businessUnitId}/weeks/{weekId}/schedules/{scheduleId}", null);
+
+        var response = await _httpClient.DeleteAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+    }
+
+    /// <inheritdoc />
+    public async Task<string> GetForecastsAsync(string businessUnitId, string weekDateId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(businessUnitId);
+        ArgumentException.ThrowIfNullOrEmpty(weekDateId);
+
+        var uri = UriHelper.GetUri($"api/v2/workforcemanagement/businessunits/{businessUnitId}/weeks/{weekDateId}/shorttermforecasts", null);
+
+        var response = await _httpClient.GetAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
+
+        return await response.Content.ReadAsStringAsync(cancellationToken);
+    }
+
+    /// <inheritdoc />
+    public async Task DeleteForecastAsync(string businessUnitId, string weekDateId, string forecastId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(businessUnitId);
+        ArgumentException.ThrowIfNullOrEmpty(weekDateId);
+        ArgumentException.ThrowIfNullOrEmpty(forecastId);
+
+        var uri = UriHelper.GetUri($"api/v2/workforcemanagement/businessunits/{businessUnitId}/weeks/{weekDateId}/shorttermforecasts/{forecastId}", null);
+
+        var response = await _httpClient.DeleteAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
```

Comment:
Instead, return IsSuccessfulStatusCode. All delete methods should return Task<bool>